src/Pimple/Container.php

```diff
@@ -279,4 +294,17 @@ public function register(ServiceProviderInterface $provider, array $values = arr
 
         return $this;
     }
+
+    /**
+     * @param $alias
+     * @param $service
+     *
+     * @return $this
+     */
+    public function alias($alias, $service)
+    {
+        $this->aliases[$alias] = $service;
```

Comment:
The first thoughts about alias was to store the aliases in an array and verify whether they exists or not. But I'm thinking about to just register a callable in the container returning the original service. What you think about it?